js/models/order/Case.js

```diff
@@ -40,6 +40,15 @@ export default class extends BaseModel {
       // convert price fields
       response.vendorContract.buyerOrder.payment.amount =
         integerToDecimal(response.vendorContract.buyerOrder.payment.amount, true);
+
+      // if (response.resolution) {
```

Comment:
This will be needed once the server is properly providing the payout amount. I'll uncomment it then.